pilot/pkg/bootstrap/options.go

```diff
@@ -151,4 +151,6 @@ func (p *PilotArgs) applyDefaults() {
 	p.KeepaliveOptions = keepalive.DefaultOption()
 	p.RegistryOptions.DistributionTrackingEnabled = features.EnableDistributionTracking
 	p.RegistryOptions.DistributionCacheRetention = features.DistributionHistoryRetention
+	p.RegistryOptions.KubeOptions.KubernetesAPIQPS = float32(features.KubernetesAPIQPS.Get())
```

Comment:
make this as cli args please rather than features

Reply:
Done